cvmfs/server/cvmfs_server_common.sh

```diff
@@ -981,6 +982,7 @@ CVMFS_HIDE_MAGIC_XATTRS=yes
 CVMFS_FOLLOW_REDIRECTS=yes
 CVMFS_SERVER_CACHE_MODE=yes
 CVMFS_NFILES=65536
+CVMFS_USE_SSL_SYSTEM_CA=true
```

Comment:
That should be added in the migration routine, too.  I think we already have the migration for repositories from 2.7 --> 2.8 in place, so it's only a matter of adding this particular parameter to existing server.conf files.

Reply:
This is still missing, I think